src/main/java/duke/TaskList.java

```diff
@@ -0,0 +1,116 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import task.Task;
+
+/**
+ * Contains the task list and operations to add/delete tasks in the list.
+ *
+ * @author felix-ong
+ */
+public class TaskList {
+    /** List of tasks */
+    private List<Task> tasks;
+
+    /**
+     * Constructor of TaskList.
+     *
+     * @param tasks The list of tasks to be used.
+     */
+    public TaskList(List<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    /**
+     * Constructor of TaskList.
+     */
+    public TaskList() {
+        this.tasks = new ArrayList<>();
+    }
+
+    /**
+     * Returns the size of the list of tasks with the appropriate grammar.
+     * @return A String describing the size of the list of tasks with the appropriate grammar.
+     */
+    public String getTaskCount() {
+        int taskCount = this.tasks.size();
+        return String.format("You have %d %s in the list.%n", taskCount, taskCount > 1 ? "tasks" : "task");
```

Comment:
Would it be less readable this way? Maybe you can try this?
```suggestion
        int taskCount = this.tasks.size();
        String name = taskCount > 1 ? "tasks" : "task";
        return String.format("You have %d %s in the list.%n", taskCount, name);
```

Reply:
I agree with @xiangjunn on this, though maybe a better variable name can be used in place of `name` (I can't think of a better name too, I used "pluralised" for mine 🤣 )